tools/topology/topology1/sof-acp-rmb-tdm8ch-dmic2ch-nau8825-max98360.m4

```diff
@@ -98,7 +98,7 @@ DAI_CONFIG(ACPHS, 1, 1, acp-amp-codec,
 # PCM Playback pipeline 4 on PCM 1 using max 2 channels of s16le.
 # 10ms deadline with priority 0 on core 0
 # this is connected to pipeline DAI 1
-PIPELINE_PCM_ADD(sof/pipe-host-playback.m4,
+PIPELINE_PCM_ADD(sof/pipe-passthrough-playback-sched.m4,
```

Comment:
This seems inconsistent with the description of pipelines

````
# PCM0 ----> buffer ------+
#                         |----> muxdemux ----> buffer ---->  ACPSP_play
# PCM1 ----> buffer ----->
````

It's very hard to review if you have the m4 stuff without updating what the topology looks like...